LBCMNoChain is deprecated in ICU 74:

From 84e47620692be90950d090f2f4722494b020ad96 Mon Sep 17 00:00:00 2001
From: Ho Cheung <uioptt24@gmail.com>
Date: Fri, 24 Mar 2023 03:54:41 +0800
Subject: [PATCH] ICU-22295 Remove deprecated RBBI LBCMNoChain option

ICU-22295 Remove deprecated RBBI LBCMNoChain option

Index: i18npool/source/breakiterator/data/line.txt
--- i18npool/source/breakiterator/data/line.txt.orig
+++ i18npool/source/breakiterator/data/line.txt
@@ -14,7 +14,8 @@
 #
 
 !!chain;
-!!LBCMNoChain;
+# DEPRECATED since ICU 74
+# !!LBCMNoChain;
 
 
 !!lookAheadHardBreak;
